assets/airports/eggw.json

```diff
@@ -1,1266 +1,1638 @@
 {
-    "_citation": "Thanks to SkyVector for coordinates and general information, NATS for providing the AIP and information for Luton and @Fechulo for the restricted areas and polygonal airspace!",
     "radio": {
         "twr": "Luton Tower",
-        "app": "Luton Approach",
-        "dep": "Luton Radar"
+        "app": "London Control",
+        "dep": "London Control"
     },
     "icao": "EGGW",
     "iata": "LTN",
-    "magnetic_north": 0.15,
-    "ctr_radius": 80,
-    "ctr_ceiling": 19500,
-    "initial_alt": 3000,
-    "position": ["N51d52m29.00", "W000d22m06.00"],
+    "magnetic_north": -0.27,
+    "ctr_radius": 90,
+    "ctr_ceiling": 28500,
+    "initial_alt": 5000,
+    "position": ["N51d52m29", "W000d22m06", "152ft"],
     "rr_radius_nm": 5.0,
-    "rr_center": ["N51d52m29.00", "W000d22m06.00"],
-    "has_terrain": false,
+    "rr_center": ["N51d52m29", "W000d22m06"],
+    "has_terrain": true,
     "wind": {
-        "angle": 285,
-        "speed": 6
+        "angle": 247,
+        "speed": 9
     },
-    "arrivalRunway": "26",
-    "departureRunway": "26",
     "airspace": [
```

Comment:
(line 21)
Would it maybe make more sense to use the North LTCC sector instead of the entire LTMA?
[TC North-West 2016-02.pdf](https://github.com/openscope/openscope/files/1735170/TC.North-West.2016-02.pdf)
[TC North-East 2016-02.pdf](https://github.com/openscope/openscope/files/1735171/TC.North-East.2016-02.pdf)
[TC North 2016_09.pdf](https://github.com/openscope/openscope/files/1735172/TC.North.2016_09.pdf)

Reply:
I'll see what I can bodge together